index.html

```diff
@@ -156,5 +156,18 @@ <h4 class="connect">LET'S CONNECT</h4>
             </div>
         </div>
     </section>
+    <div id="class_background">
+        <section class="contactMe" id="contact_back">
+            <h2>Contact me</h2>
+            <p class="contact_info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I'd love to help with it</p>
+            <form action="">
+                <input required type="text" name="user_name" maxlength="30" placeholder="Your name" class="form_info">
+                <input required type="mail" name="user_mail" placeholder="Your email" class="form_info">
+                <textarea required name="user_message" id="user_message" maxlength="500" placeholder="Write your message here..."></textarea>
+                <button class="contact_button" type="button">Get in touch</button>
+            </form>
+        </section>
```

Comment:
- You've done an amazing job here 👍  but there are a few things that I want to suggest to increase your user experience ➕ , design, and to the Figma wireframe which is the requirement. ✍️ 
- [ ] In the schedule of images I will show you now, you notice that your inputs need some `padding` for spacing so users can see what they're typing clearly. This will enhance user experience 😉 👍 
- [ ] Also, the button states (hover and active) are not applying to your button here as against convention. Kindly add the respective button states like other buttons. To keep your code DRY, you consider using the same class to effect similar styles so that you don't repeat some codes. 💪 🔧

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/220489389-775ca0e2-8dfc-4bef-b2f6-81709fc8ac83.png)|![image](https://user-images.githubusercontent.com/99715640/220489428-db038317-ac98-4084-a43b-56248152a188.png)|